src/components/Hero/ContentHero/index.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react"
+import { Box, Heading, HStack, SimpleGrid, Stack, Text } from "@chakra-ui/react"
+import Breadcrumbs, { IProps as BreadcrumbsProps } from "../../Breadcrumbs"
+import { GatsbyImage } from "../GatsbyImage"
+import { CallToAction } from "../CallToAction"
+import { CommonHeroProps } from "../utils"
+
+export interface ContentHeroProps extends CommonHeroProps {
+  breadcrumbSlug: BreadcrumbsProps["slug"]
```

Comment:
To be consistent with the MdxHero api, I'd call this `breadcrumb` and accept any breadcrumb prop.

```tsx
// and then just
<Breadcrumbs {...breadcrumbs} />
```